packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -9,8 +9,11 @@ import UnitRowWithAvatar from '../UnitRowWithAvatar';
 import Modal from '../Modal';
 import AlertBar from '../AlertBar';
 import { AccountData } from '../AccountDataHOC/gql';
+import { RouteComponentProps } from '@reach/router';
```

Comment:
I was complaining about something similar to @LZoog earlier today, so maybe this could be in a future refactor, but I think if we add `@reach/router` to the tsconfig's `typeRoots` we should be able to avoid needing to import type defs. I think 🤔

Reply:
There might be a better way to do this but it was mentioned in the react reach docs. Maybe its ok for now?

Reply:
Oh yeah it's totally fine. :)